anchore_engine/clients/grype_wrapper.py

```diff
@@ -66,12 +94,19 @@ class GrypeWrapperSingleton(object):
     # These values should be treated as constants, and will not be changed by the functions below
```

Comment:
will be nice at some point to have a docstring calling out differences between staging and prod grype DB